program/index.html

```diff
@@ -0,0 +1,220 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Выбор программы</title>
+  </head>
+  <body>
+    <main class="main">
+      <section class="header container">
+        <img
+          class="header__logo"
+          src="../img/logo_car_energy.svg"
+          alt="logo_car_energy"
+          width="202"
+          height="60"
+          alt="logo_car_energy"
+        />
+        <ul class="header__menu">
+          <li class="header__item">
+            <a class="header_link" href="#">Главная</a>
+          </li>
+          <li class="header__item">
+            <a class="header_link" href="#">Каталог продукции</a>
+          </li>
+          <li class="header__item">
+            <a class="header_link" href="#">Выбор программы</a>
+          </li>
+        </ul>
+      </section>
+
+      <div class="select-program">
+        <h2 class="select-program__title title">Подбор программы</h2>
+        <form class="" action="" method="GET">
+          <legend class="select-program__decription">
+            Заполните анкету и мы подберем программу питания для вашего кота
+          </legend>
+          <ul class="select-program__data" role="none">
+            <li class="select-program__item">
+              <label class="select-program__text" for="name">Имя:*</label>
+              <input id="name" type="text" name="name" />
+            </li>
+            <li class="select-program__item">
+              <label class="select-program__text" for="weight"
+                >Вес (кг):*</label
+              >
+              <input id="weight" type="text" name="weight" />
```

Comment:
Input type should be `number` for number values, `email` for emails, `tel` for phones, etc. Please use a corresponding type — this is necessary because on a smartphone a needed type of keyboard will be opened automatically when a field is focused. Numeric for numbers, optimized for email entering for email fields, and so on.

Reply:
fix